### DISCUSSION TITLE
basic-extra-meal/enum

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#296] [Announcements]
# basic-extra-meal/enum

本文全面介绍了Java枚举的概念、基础语法、高级应用以及在实际项目中的应用。通过详细的示例和解释，帮助读者深入理解枚举类型的使用

https://javabetter.cn/basic-extra-meal/enum.html

**Comment by fusionlin** [comment]
编译后的字节码，你是如何得到的，怎么和我使用javap工具或idea自带的showByteCode获取的不一样？
